backend/modules/invoices.js

```diff
@@ -63,8 +84,49 @@ router.get('/file/:id', async (req, res) => {
     res.send(file);
   } catch (e) {
     console.error(e);
-    res.status(500).json({success: false, message: 'Internal Server Error'});
+    res.status(500).json({ success: false, message: 'Internal Server Error !' });
   }
 });
 
+// Endpoint to add a new invoice
+app.post('/invoice/new', checkRole, (req, res) => {
+  const { amount, eventId } = req.query;
+
+  // amount and eventId are necessary
+  if (!amount || !eventId) {
+    return res.status(400).json({ error: 'amount and eventId are required in query parameters' });
```

Comment:
Make sure that all the return responses have a `success` and `message` keys. Refer to backend README.md file for the return object structure.
You can also drop the `error` key, and rename it to `message` instead.